.github/workflows/move2kube-e2e.yaml

```diff
@@ -2,23 +2,26 @@ name: Move2kube Workflow end to end tests
 
 on:
   workflow_dispatch:
-  pull_request_target:
```

Comment:
whyyyyyyyyyyyyy
https://www.codu.co/articles/when-to-use-pull_request_target-instead-of-pull_request-in-github-actions-k3pao1qz